### ISSUE TITLE
A11y accessibility features

### ISSUE THREAD

**Issue by micheledallatorre** [micheledallatorre/mws-restaurant-stage-1#5]
Implement accessibility features in the site HTML:
  - [x] set tabIndex where missing on tables, links, etc.
  - [x] mark maps as `role="application" aria-hidden="true"`
  - [x] add `aria-label` on (select) buttons with no labels
  - [x] on nav, add `aria-label` and `aria-current="page"`
  - [x] fix links without no or wrong href
  - [x] add alt text for images
  - [ ] skip links?
  - [ ] tabfocus?
  - [x] check h1 and h2, etc. are correctly set up in #3
